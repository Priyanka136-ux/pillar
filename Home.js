import React from 'react';
import { View, StyleSheet, Button } from 'react-native';


const styles = StyleSheet.create({
    outerContainer: {
        flex: 1,
        marginTop: 100
    },
    container: {
        flex: 1,
        margin: 25,
        backgroundColor: "white"
    },
    button: {
        width: "50%",
        color: "orangered",
        marginTop: 100,
        marginBottom: 50,
        marginLeft: 80
    },

    buttonSecondary: {
        width: "50%",
        color: "orangered",
        flex: 1,
        marginBottom: 50,
        marginLeft: 80
    }

})
const Home = ({ navigation }) => {

    return (
        <View style={styles.container}>
            <View style={styles.button}>
                <Button onPress={() => navigation.navigate('Mnemonic')} title="Save" />
            </View>
            <View style={styles.buttonSecondary}>
                <Button onPress={() => navigation.navigate('Transaction')} title="Balance/Transaction" />
            </View>
        </View>
    )
}

export default Home;

