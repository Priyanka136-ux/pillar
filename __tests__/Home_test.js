

import 'react-native';
import renderer from 'react-test-renderer';
import React from 'react';
import Home from '../Home';
import {jest} from '@jest/globals';

 jest.useFakeTimers();

test('renders correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
