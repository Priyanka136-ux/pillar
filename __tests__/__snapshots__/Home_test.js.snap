// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
      "margin": 25,
    }
  }
>
  <View
    style={
      Object {
        "color": "orangered",
        "marginBottom": 50,
        "marginLeft": 80,
        "marginTop": 100,
        "width": "50%",
      }
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          Save
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "color": "orangered",
        "flex": 1,
        "marginBottom": 50,
        "marginLeft": 80,
        "width": "50%",
      }
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          Balance/Transaction
        </Text>
      </View>
    </View>
  </View>
</View>
`;
